### PR TITLE
Updated setup.py to make it build with python 3.7.

### PR DESCRIPTION
**Pull request by RatanShreshtha**

### CODE REVIEW

**Comment by pquentin** [comment]
Thanks!

The build is going to fail because we don't have https://github.com/urllib3/urllib3/pull/1457 yet, so I'll add Python 3.7 to the build matrix later.

The changes look good, thanks, but can you restore the comment? You can also fix the typo: Remove -> Removing while you're at it.

**Comment by RatanShreshtha** [comment]
Yeah sure I will do that

File: setup.py [hunk 1]
```diff
@@ -60,8 +60,8 @@ def bleach_tokens(tokens):
     # TODO __await__, ...?
     used_space = None
     for space, toknum, tokval in tokens:
-        if toknum in [ASYNC, AWAIT]:  # TODO Python 3.7+
-            # When remove async or await, we want to use the whitespace that
+        if tokval in ["async", "await"]: # TODO Python 3.7+
```
Comment (pquentin) [inline setup.py]: Sorry if I was not clear. You need to remove the "# TODO Python 3.7+" comment, but not the following lines.

**Comment by pquentin** [comment]
Thank you @RatanShreshtha for your contribution! Merged.

**Pull request merged**